code/_onclick/observer.dm

```diff
@@ -66,12 +66,21 @@
 	if(target)
 		user.loc = get_turf(target)
 
-/obj/machinery/gateway/centerstation/attack_ghost(mob/user as mob)
-	if(awaygate)
-		user.loc = awaygate.loc
+// VOREStation Edit Begin
+
+/obj/machinery/gateway/centerstation/attack_ghost(mob/user as mob) 
+	if(awaygate) 
+		if(user.client.holder) 
+			user.loc = awaygate.loc
+		else if(active)
+			user.loc = awaygate.loc
+		else
+			return
```

Comment:
This should probably have a to_chat instead of a return, similar to the line below this. Probably "[src] is not powered up."

Reply:
Ghosts interacting with things they're not able to typically doesn't give a message, to my knowledge. Giving a message here would be a bit odd.

Reply:
...Good point.